mollie.php

```diff
@@ -274,6 +274,8 @@ public function getContent()
 
         Media::addJsDef([
             'description_message' => addslashes($this->l('Enter a description')),
+            'min_amount_message' => addslashes($this->l('You have entered incorrect min amount')),
+            'max_amount_message' => addslashes($this->l('You have entered incorrect max amount')),
```

Comment:
MediaUtility::scapeStrings([
])

you can create service which iterates over array , and for strings addshales , and I think it should do it recursivly.

not mandatory to work with that atm